02_Development/Display/features/environment.py

```diff
@@ -0,0 +1,8 @@
+
+
```

Comment:
should there be a before_step to start the container?

Reply:
like, a before_all to start the container, and after_all to kill the container if it's not already dead?

Reply:
I think this is an instance of github being bad.  I've added that